Documentation/troubleshooting.rst

```diff
@@ -180,6 +180,35 @@ both to the node itself and to an endpoint on that node. The latency specified
 is a snapshot at the last time a probe was run, which is typically once per
 minute.
 
+Kubernetes
+==========
+
+Pod not managed by Cilium
+-------------------------
+
+In some situations, Cilium is not managing networking of a pod. These
+situations are:
+
+* The pod is running in host networking and will use the host's IP address
+  directly. Such pods have full network connectivity but Cilium will not
+  provide security policy enforcement for such pods.
+
+* The pod was started before Cilium was deployed. Cilium only manages pods
+  that have been deployed after Cilium itself was started. Cilium will not
+  provide security policy enforcement for such pods.
+
```

Comment:
We also annotate the pod with a `cilium-identity` so that is another way to see this (I don't know the jsonpath to do this but it would be a kubectl one-liner).

Reply:
Good point. What do you consider more up to date?

Reply:
The CEPs, nominally, since they run every 10s. In this case, though, there is a subtle distinction given our recent bug reports: the CEP will tell you what the current cilium agent is managing (the background garbage collection will catch anything staler than a few minutes). The annotation tells you if cilium ever tried to manage the pod (and got as far as giving it an identity). It isn't that important though, and I couldn't figure out the jsonpath to make the annotation lookup as simple as your script.